modules/infrastructure-moving-logging.adoc

```diff
@@ -3,10 +3,84 @@
 // * machine_management/creating-infrastructure-machinesets.adoc
 
 [id="infrastructure-moving-logging_{context}"]
-= Moving the logging aggregation solution
+= Moving the cluster logging resources
 
-[IMPORTANT]
+You can configure the Cluster Logging Operator to deploy the pods 
+for any or all of the Cluster Logging components, Elasticsearch, Kibana, Curator, and 
+Fluentd, to different nodes. You cannot move the Cluster Logging Operator pod from its installed location. 
+
+[NOTE]
 ====
-Cluster logging in {product-title} is not installed by default.
-You currently cannot move the logging component to another MachineSet.
+You should set your MachineSet to use at least 6 replicas.
 ====
+
+.Prerequisite
+
+* Cluster logging and Elasticsearch must be installed. These features are not installed by default.
```

Comment:
Logging

Reply:
We have been using `cluster logging` in lowercase when talking about the feature in general.